content/learn/advanced-examples/sudoku/_index.md

```diff
@@ -1,5 +1,8 @@
 +++
 title = "Sudoku"
-weight = 2
 template = "docs-section.html"
+page_template = "docs-page.html"
```

Comment:
I may be wrong, but isn't `page_template` inherited? Since it is set in `advanced-examples/_index.md`, does it also need to be set here?

Reply:
Honestly not sure. I'll do some tests later to check. Hmm.

Reply:
Ope, yep. I'll remove all the unnecessary `page_template`'s across the content then.